src/features/defi/providers/fox-farming/components/FoxFarmingManager/Claim/ClaimStatus.tsx

```diff
@@ -217,31 +275,21 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({ accountId }) => {
           </Row>
           <Row>
             <Row.Label>
-              {translate(
-                state.txStatus === TxStatus.PENDING
-                  ? 'modals.status.estimatedGas'
-                  : 'modals.status.gasUsed',
-              )}
+              {translate(status ? 'modals.status.estimatedGas' : 'modals.status.gasUsed')}
```

Comment:
Are we intentionally changing the logic here such that _any_ status will use the `modals.status.estimatedGas` translation?

Reply:
Nice catch - that wasn't intentional indeed! 82e3beffa5